src/style.css

```diff
@@ -85,4 +85,10 @@ li[aria-hidden="true"] {
   padding: 0;
   position: absolute;
   width: 1px;
+}
+
+/* added to better visualize the subtitle and the User Guide link */
+h2 {
```

Comment:
Is making it responsive another task or do you do it within this one?
Currently it looks like this 👇🏽 
<img width="542" alt="Screenshot 2023-11-07 at 08 12 10" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/54622834/37211d17-d429-48eb-bb4f-0c8b0f057f60">

I would also suggest having the second sentence constantly in a new line, by e.g. adding `<br />`.
A line-height should also be added. I think 1.5 is standard for accessibility in headings - should be looked up again to make sure.

Reply:
We should also think of a appropriate line break in `h1`, e.g. being a one-linee but as soon as it breaks the heading should look like

> Tagged!
> The HTML Quiz

or so, nothing in between.

Reply:
It looked all over the place on my desktop so I just wanted to center it at least, but to be honest I was a bit at a loss about the best way to make it look appealing. I really like your suggestion about the h1. Same idea about the subheading?

Happy to implement the styling now, or leave it for later when we'll approach the overall styling.

Reply:
Yeah, I think a new issue for that would be the better choice since this one was only on what the text should contain.
New issue should/could contain: forced line break to both, h1 and h2, minimum line height as suggested by w3, text align adjustment (removing width on h2 as it seems, and not exactly sure was margin-inline does).

Reply:
Sounds good! I updated the PR description and removed the styling I added. I shouldn't have added it to begin with, sorry for the extra trouble! 

(Margin-inline is a shortcut for the left and right margins.)

Reply:
No worries 😄 

Margin-inline: Yeah I thought so but not sure if it had an effect since the container has `text-align: center` set already.